src/config.xml

```diff
@@ -3802,6 +3802,130 @@ where `loc1` and `loc2` can be relative or absolute paths or URLs.
 ]]>
       </docs>
     </option>
+    <option type='list' id='DOT_GRAPH_ATTR' format='string' depends='HAVE_DOT'
+      defval='bgcolor=\"transparent\"'>
+      <docs>
+<![CDATA[
+ \c DOT_GRAPH_ATTR is common attributes for graphs.
+ For details please see
+ <a href="https://graphviz.org/doc/info/attrs.html">Node, Edge and Graph Attributes specification</a>.
+ <br>The default value is:<br>
```

Comment:
Text for the default value is generated automatically, so sentence should not be here.

Reply:
Except when I run `doxygen -g`, the sentence isn't generated but the value is inserted (with a period after and no space or new line).

Reply:
I think the behavior is different since this is `type='list'` not `type='string'`.